### ISSUE TITLE
Include Level0 in the edit menu

### ISSUE THREAD

**Issue by grischard** [openstreetmap/openstreetmap-website#2391]
Level0 is perfect for quick small corrections. In the edit drop-down, Level0 should be included. If that's too dangerous for newbies, maybe it can be a checkbox in the settings.

**Comment by tomhughes** [comment]
Well there are two issues, firstly the fact that it's not exactly what you'd call user friendly and secondly that of baking in dependencies on third party sites.

**Comment by woodpeck** [comment]
In my opinion, someone who is knowledgeable and mature enough to use Level0 without making an undiscussed mechanical edit will also be able to add themselves a Level0 bookmarklet or Greasemonkey script. Actually "looking at the map" while editing is something that avoids a ton of pitfalls, and I would really like to avoid officially suggesting that Level0 is a good way to make edits.

**Comment by dieterdreist** [comment]
or an edit button, so you could add or modify tags of existing objects „without an editor“, right in the osm map sidebar/detail view. For  just-fixing-a-typo edits this would often be useful.

level0 actually does provide a map view, although it didn’t work for me on a mobile phone (edited a road in Italy and the marker was somewhere in Algeria). I just used level0 and agree it is kind of cumbersome to copy the id, prepend a w/n/r before pasting, rather than using the editor mechanism on the webpage, but I guess it won’t be disputed that level0 is generally only suitable for experienced mappers, so maybe don’t bother beginners with this choice?

**Comment by mmd-osm** [comment]
@grischard : I'm curious what particular kind of correction you had in mind, and how you would have done it in Id or JOSM.

**Comment by Zverik** [comment]
@dieterdreist you don't have to copy and edit an id — the full URL would work.

If we decide on somehow referencing Level0 from the website, I'd prefer it to be a like near "download XML" on node/way/relation panels.

**Comment by tordans** [comment]
> Should we close this in favor of #3611?

I don't think so. They are very different things, IMO. We will have a policy on how to add new entries to the edit menu in the near future. But planning and maintaining features like an editor inside the website is something else entirely.

**Comment by mmd-osm** [comment]
I thought more about the use case of doing "quick small corrections", rather than the technical details of where this is implemented (part of the website or linking to external sites). If these small corrections could be covered by a simple tag editor that is less difficult to work with for "normal" users, then our requirement may be sufficiently covered already.
